File: 0412-fizz-buzz/0412-fizz-buzz.go
import "strconv"

func fizzBuzz(n int) []string {
	slice := []string{}
	for i := 1; i <= n; i++ {
		if (i%5 == 0) && (i%3 == 0) {
			slice = append(slice, "FizzBuzz")
		} else if i%5 == 0 {
			slice = append(slice, "Buzz")
		} else if i%3 == 0 {
			slice = append(slice, "Fizz")
		} else {
			slice = append(slice, strconv.Itoa(i))
		}
	}
	return slice
}

File: 0412-fizz-buzz/0412-fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        
        arr = []
        
        for i in range(1, n+ 1):
            
            if not i % 3 and not i % 5:
                
                arr.append("FizzBuzz")
                
            elif not i % 5:
                arr.append("Buzz")
            
            elif not i % 3:
                arr.append("Fizz")
            
            else:
                arr.append(str(i))
        
        return arr
                           